src/Kunstmaan/MediaBundle/Entity/Media.php

```diff
@@ -11,8 +11,8 @@
  *
  * @ORM\Entity(repositoryClass="Kunstmaan\MediaBundle\Repository\MediaRepository")
  * @ORM\Table(name="kuma_media", indexes={
- *      @ORM\Index(name="idx_media_name", columns={"name"}),
- *      @ORM\Index(name="idx_media_deleted", columns={"deleted"})
+ *      @ORM\Index(name="idx_name", columns={"name"}),
```

Comment:
I think this went wrong in a merge conflict? it should be idx_media_name and idx_media_deleted because of 07768d058d78b42839c78eb586cc025ce166eada